code/src/components/Loader.css

```diff
@@ -0,0 +1,90 @@
+.loader-overlay {
+  display: flex;
+  flex-direction: column;
+	position: absolute;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	background: #000;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.loader-overlay h1 {
+    color: #fff;
+}
+
+.loading-heart {
+    position: relative;
+    width: 100px;
+    height: 90px;
+    animation: loaderheartbeat 2s infinite;
+    margin:0 auto
+  }
+  .loading-heart:before,
+  .loading-heart:after {
+    position: absolute;
+    content: "";
+    left: 50px;
+    top: 0;
+    width: 50px;
+    height: 80px;
+    background: #fff;
+    -moz-border-radius: 50px 50px 0 0;
+    border-radius: 50px 50px 0 0;
+    -webkit-transform: rotate(-45deg);
+       -moz-transform: rotate(-45deg);
```

Comment:
I think most of the modern browsers support regular selectors, so we don't have to specify -webkit-, -moz-, -ms- and -o- anymore.  For example for the transform selector, we just need to write `transform: rotate(-45deg); `

Reply:
Oh good to know! Thank you!